File: Advanced Proxy check/Advanced_Proxy_Check.py
import tkinter as tk
import requests
from bs4 import BeautifulSoup

# Define the function to check the proxy
def check_proxy():
    # Get the proxy from the entry field
    proxy = proxy_entry.get()

    # Send a GET request to the IPVoid proxy checker with the proxy as a parameter
    url = f'https://www.ipvoid.com/proxy-check/{proxy}/'
    response = requests.get(url)

    # Extract the response data using BeautifulSoup
    soup = BeautifulSoup(response.text, 'html.parser')
    result = soup.find_all('div', {'class': 'panel-body'})[1].text.strip()

    # Display the result in the label
    result_label.config(text=result)

# Create the GUI window
root = tk.Tk()
root.title("Advanced Proxy Check")

# Create the label and entry field for the proxy
proxy_label = tk.Label(root, text="Proxy:")
proxy_label.pack(side=tk.LEFT)
proxy_entry = tk.Entry(root)
proxy_entry.pack(side=tk.LEFT)

# Create the button to check the proxy
check_button = tk.Button(root, text="Check", command=check_proxy)
check_button.pack(side=tk.LEFT)

# Create the label to display the result
result_label = tk.Label(root, text="")
result_label.pack(side=tk.BOTTOM)

# Start the GUI event loop
root.mainloop()

File: Advanced Proxy check/Advanced_Proxy_Check.java
import javax.swing.*;
import java.awt.*;
import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;
import java.io.IOException;
import org.jsoup.Jsoup;
import org.jsoup.nodes.Document;
import org.jsoup.nodes.Element;

public class ProxyChecker {
    public static void main(String[] args) {
        SwingUtilities.invokeLater(() -> {
            createAndShowGUI();
        });
    }

    private static void createAndShowGUI() {
        JFrame frame = new JFrame("Advanced Proxy Check");
        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        frame.setSize(400, 150);
        frame.setLayout(new FlowLayout());

        JLabel proxyLabel = new JLabel("Proxy:");
        JTextField proxyEntry = new JTextField(20);

        JButton checkButton = new JButton("Check");
        JLabel resultLabel = new JLabel();

        checkButton.addActionListener(new ActionListener() {
            @Override
            public void actionPerformed(ActionEvent e) {
                String proxy = proxyEntry.getText();
                String result = checkProxy(proxy);
                resultLabel.setText(result);
            }
        });

        frame.add(proxyLabel);
        frame.add(proxyEntry);
        frame.add(checkButton);
        frame.add(resultLabel);

        frame.setVisible(true);
    }

    private static String checkProxy(String proxy) {
        try {
            String url = "https://www.ipvoid.com/proxy-check/" + proxy + "/";
            Document doc = Jsoup.connect(url).get();
            Element resultElement = doc.select("div.panel-body").get(1);
            return resultElement.text().trim();
        } catch (IOException e) {
            return "Error checking proxy.";
        }
    }
}
